src/features/dashboard/DashboardPage.tsx

```diff
@@ -81,6 +81,8 @@ export default function DashboardPage(): JSX.Element {
   const [cards, setCards] = useState<TNFTCard[]>([])
   const [tab, setTab] = useState<number>(0)
   const [openNotificationModal, setOpenNotificationModal] = useState(false)
+  const [walletBalance, setWalletBalance] = useState(0)
+  const [transactions, seWTransactions] = useState<TTransactionItemProps[]>([])
```

Comment:
Correct name `seWTransactions` to `setTransactions`

Reply:
Updated